build_daemon/pubspec.yaml

```diff
@@ -26,4 +26,4 @@ dev_dependencies:
   mockito: ^4.0.0
   test: ^1.3.3
   test_descriptor: ^1.1.1
-  uuid: ^1.0.3
```

Comment:
Given that all the casts are actually in test code only, we could leave this at `>=1.0.3 <3.0.0`

Reply:
Does this package have enforced static analysis in presubmits? bumping to uuid 2.0.0 should introduce UNNECESSARY_CAST Hints.

Reply:
It does have unnecessary cast hints. Since this is only a dev dep though it should be fine to bump the minimum.